vocabularies/UI.xml

```diff
@@ -1889,6 +1889,51 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         </Property>
       </ComplexType>
 
+      <Term Name="Recommendations" Type="Edm.ComplexType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Recommendations for an entity" />
+          <Annotation Term="Core.LongDescription">
+          <String>This complex-typed annotation contains structural properties corresponding via name equality
+to non-key structural properties of the entity type for which recommendations are available.
+The type of such a property is a collection of an informal specialization of [`PropertyRecommendationType`](#PropertyRecommendationType).
+
+Clients retrieve the recommendations with a GET request that includes this annotation in a `$select` clause.
+The recommendations SHOULD be computed asynchronously, see [this diagram](../docs/recommendations.md).</String>
+          </Annotation>
+      </Term>
+      <ComplexType Name="PropertyRecommendationType" Abstract="true">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Base type containing recommendations for an entity type property" />
+        <Property Name="RecommendedFieldValue" Type="Edm.PrimitiveType" Nullable="false">
+          <Annotation Term="Core.Description" String="Recommended value" />
+          <Annotation Term="Core.LongDescription">
+            <String>In informal specializations of this base type, this property is specialized to the primitive type of the entity type property.
+            If the recommendation has a description, this property has a [`Common.Text`](Common.md#Text) annotation
+            that evaluates to the `RecommendedFieldDescription` property.</String>
+          </Annotation>
+          <Annotation Term="Common.Text" Path="RecommendedFieldDescription" />
+        </Property>
+        <Property Name="RecommendedFieldDescription" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Description of the recommended value" />
+          <Annotation Term="Core.LongDescription">
+            <String>In informal specializations of this base type, this property is specialized to the string type of the text property corresponding to the entity type property.
+            It is omitted from informal specializations for recommendations without description.</String>
```

Comment:
Ah, finally the explanation for "informal". Please also state with the first mentioning of "informal".